### ISSUE TITLE
[TBC Bugs Draenei Leveling Phase - Kessel Run Mount should not be usable in Bloodmyst areas]

### ISSUE THREAD

**Issue by Paradigmxd** [chromiecraft/chromiecraft#5409]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

Generic

### Current Behaviour

You can use the Mount from Kessel Run to go to the Village in Bloodmyst Isle...

### Expected Blizzlike Behaviour

Mount should NOT be usable as described in the evidence section 

ADditionally, text should appear from Kessel saying something like 'You are going too far traveller' or something and be dismounted

### Source

https://wowpedia.fandom.com/wiki/The_Kessel_Run

### Steps to reproduce the problem

DO the quest and use the mount to go to the Quest Hub vIllage in Bloodmyst Isle

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ce6ca597b54e64f328cacddae5f003cdc41a343d

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Rorschach91** [comment]
Ok, I checked on official server and there are my conclusions.

In the official servers you can go everywhere with Kessel's mount, except for the Bloodmyst Isle's part over the river.... Or rather, that is how it should be.

There is a trigger on the bridge that generates a dialogue and dismount you.
 
![World Of Warcraft Screenshot 2024 08 18 - 21 20 21 67](https://github.com/user-attachments/assets/9eb0af2a-b593-4b09-aff3-c5f6aee9cec5)

But now comes the fun part:

That trigger is ONLY present on the bridge. This means that if you cross the river by the water, trigger doesn't activate and you are able to go where you prefer.

![World Of Warcraft Screenshot 2024 08 18 - 21 13 33 71](https://github.com/user-attachments/assets/0b87be7c-ba0f-4801-a7b7-a454d920184e)

It is obvious that this was not intended by the Blizzard, but this is the offy.
In any case you haven't restrictions in Azuremyst Isle. You can even get inside The Exodus:

![World Of Warcraft Screenshot 2024 08 18 - 21 54 53 58](https://github.com/user-attachments/assets/1d7a40af-8e40-4108-965f-9b772e702da3)

In conclusion: Only the bridge trigger is missing on Chromiecraft.

Thank you for your issue. If you find other bugs feel free to report them.
